openedx/core/djangoapps/external_auth/tests/test_shib.py

```diff
@@ -332,21 +331,23 @@ def test_registration_form_submit(self, identity):
         self.assertEquals(len(audit_log_calls), 3)
         method_name, args, _kwargs = audit_log_calls[0]
         self.assertEquals(method_name, 'info')
-        self.assertEquals(len(args), 1)
-        self.assertIn(u'Login success on new account creation', args[0])
-        self.assertIn(u'post_username', args[0])
-        method_name, args, _kwargs = audit_log_calls[1]
-        self.assertEquals(method_name, 'info')
         self.assertEquals(len(args), 2)
         self.assertIn(u'User registered with external_auth', args[0])
         self.assertEquals(u'post_username', args[1])
-        method_name, args, _kwargs = audit_log_calls[2]
+
+        method_name, args, _kwargs = audit_log_calls[1]
```

Comment:
The order of these logging statements was changed by moving the code around above, this just updates the indexes to match.

Reply:
:+1:

Am I also reading that this test was skipped under 1.11?

Reply:
It was marked for skipping early on, but skipping was never added. PLAT-1880 exists to remove all of the tagging, but I just remove them whenever I see them.